README.md

```diff
@@ -1301,6 +1310,203 @@ By default, it will revert all the clusters present in the backup folder
+
+### Authentication Issues
+
+**Account Level:** Not only workspace, but account level authentication
+is also needed for installing UCX. If you do not have an account
+configured in .databrickscfg, you will get an error message
+“.databrickscfg does not contain account profiles; please create one
+first”.
+
+**Solution:** To authenticate with a Databricks account, consider using
+one of the following authentication types: [<u>OAuth machine-to-machine
+(M2M)
+authentication</u>](https://docs.databricks.com/en/dev-tools/cli/authentication.html#m2m-auth),
+[<u>OAuth user-to-machine (U2M)
+authentication</u>](https://docs.databricks.com/en/dev-tools/cli/authentication.html#u2m-auth),
+[<u>Basic authentication
+(legacy)</u>](https://docs.databricks.com/en/dev-tools/cli/authentication.html#basic-auth).
+
+### Workspace has an external Hive Metastore (HMS)
+
+**External HMS connectivity from UCX clusters:** If the workspace has an
+external HMS, the clusters running the UCX jobs need to have specific
+configurations to connect to the external HMS. Otherwise, UCX assessment
+will not be able to assess the tables on HMS.
+
+**Solution:** Use a cluster policy before installation to set the
+required Spark config for connecting to the external HMS, or manually
+edit the cluster post-installation to have the correct configurations.
+Detailed steps can be found
+[<u>here</u>](https://github.com/databrickslabs/ucx/blob/main/docs/external_hms_glue.md).
+
+**External HMS connectivity from UCX SQL warehouse:** UCX requires a SQL
+warehouse to create tables, run queries, create and refresh dashboards.
+If you already have a Pro or Serverless warehouse connected to the
+external HMS, you can select the same warehouse for UCX. You will also
+be given an option (during installation) to create a new warehouse for
+UCX. If you have never used a warehouse before, the new warehouse
+created might not have proper configuration set to connect to the
+external HMS.
+
+**Solution:** Set Spark configuration for connecting to external HMS in
+the Admin Settings of SQL warehouse. This will only be needed if the
+admin settings do not have the configurations already set. For example,
+add *spark.hadoop.javax.jdo.option.ConnectionURL \<connectionstring\>*
+under Data Access Configuration of SQL Warehouse Admin Settings.
+
+# Verify the Installation
+
+Once the UCX command “*databricks labs install ucx*” has completed
+successfully, the installation can be verified with the following steps:
+
+1.  Go to the Databricks Catalog Explorer and check if a new schema for
+    ucx is available in Hive Metastore with all empty tables.
+
+2.  Check that the UCX jobs are visible under Workflows.
+
+3.  Run the assessment. This will start the UCX clusters, crawl through
+    the workspace, and display results in the UCX dashboards. In case of
+    external HMS, verify from the results that the assessment has
+    analyzed the external HMS tables.
+
+# Contact Us
+
+For any concerns/issues:
```

Comment:
Maybe add a link to the contributing guide for advanced instructions

Reply:
Removed this section.

Reply:
Added reference to the troubleshooting guide